apps/web/pages/_app.tsx

```diff
@@ -49,7 +57,7 @@ const connectors = connectorsForWallets(
 
 const config = createConfig({
   connectors,
-  chains: [mainnet, base, sepolia, baseSepolia],
+  chains: [mainnet, baseSepolia, sepolia, base],
```

Comment:
does the order change do anything here?

Reply:
I don't believe it does, changed it back to the original format